README.md

```diff
@@ -117,5 +117,5 @@ run on `localhost:3001`
 Build lib bundle to `dist` folder.
 
 ```bash
-npm run build
+yarn run build
```

Comment:
There is no need to add `run` before command when using `yarn`. Just same as `yarn start` above.